specs/phase0/beacon-chain.md

```diff
@@ -1378,8 +1397,12 @@ def get_attestation_component_deltas(state: BeaconState,
     for index in get_eligible_validator_indices(state):
         if index in unslashed_attesting_indices:
             increment = EFFECTIVE_BALANCE_INCREMENT  # Factored out from balance totals to avoid uint64 overflow
-            reward_numerator = get_base_reward(state, index) * (attesting_balance // increment)
-            rewards[index] += reward_numerator // (total_balance // increment)
+            if in_inactivity_leak(state):
+                # Full base reward will be cancelled out by inactivity penalty deltas
```

Comment:
This magical line may worth more comments
 
```suggestion
                # Since full base reward will be canceled out by inactivity penalty deltas,
                # the attester that participated optimally gets full base reward compensation here.
```